src/Symfony/Component/Console/Cursor.php

```diff
@@ -183,8 +183,8 @@ public function getCurrentPosition(): array
     {
         static $isTtySupported;
 
-        if (null === $isTtySupported && \function_exists('proc_open')) {
-            $isTtySupported = (bool) @proc_open('echo 1 >/dev/null', [['file', '/dev/tty', 'r'], ['file', '/dev/tty', 'w'], ['file', '/dev/tty', 'w']], $pipes);
+        if (null === $isTtySupported) {
+            $isTtySupported = ('/' === \DIRECTORY_SEPARATOR && stream_isatty(\STDOUT));
```

Comment:
IIUC, `stream_isatty` supports Windows, so do we need the `'/' === \DIRECTORY_SEPARATOR` part?

Reply:
Windows does not support `echo 1 >/dev/null`, so we need `'/' === \DIRECTORY_SEPARATOR`

Reply:
I don't get it, sorry. Are you saying that `stream_isatty` is using `echo 1 >/dev/null` behind the scenes?

Reply:
To be clear, I think having `$isTtySupported = stream_isatty(\STDOUT);` is enough.

Reply:
`getCurrentPosition()` does not support in windows.

`stream_isatty(\STDOUT)` may also return `true` in windows.

php:
```php
$cursor = new Cursor($output);
var_dump($cursor->getCurrentPosition());
```

log:
```log
'stty' is not recognized as an internal or external command,
operable program or batch file.
'stty' is not recognized as an internal or external command,
operable program or batch file.

'stty' is not recognized as an internal or external command,
operable program or batch file.
array(2) {
  [0]=>
  NULL
  [1]=>
  NULL
}
```